service/web/server_test.go

```diff
@@ -103,3 +104,32 @@ func TestGetProjectIDFromRequest(t *testing.T) {
 		assert.Equal(t, tst.err, err)
 	}
 }
+
+func Test_getResourceNameFromRequest(t *testing.T) {
+	type test struct {
+		request interface{}
+		name    string
+		err     error
+	}
+	type testReq struct {
+	}
+	tests := []test{
+		{
+			request: &metrov1.GetSubscriptionRequest{
```

Comment:
@sumitkrzp we can also add description about each test case.

Reply:
added comments for the test cases